package.json

```diff
@@ -13,18 +13,26 @@
     "url": "https://github.com/plotly/react-plotly.js-editor/issues"
   },
   "scripts": {
+    "combine-translation-strings": "babel-node utils/combineTranslationKeys.js",
     "find-strings": "babel-node utils/findLocaleStrings.js",
```

Comment:
actually find-strings could be changed to `npm run make:translations`

Reply:
or better
`npm run make:locale` or `npm run make:localization`

Reply:
we're not making translations or localizations, we're making the list of strings from which someone would make the required translations

Reply:
or whatever. `make:something` ;)

Reply:
`make:translation-strings`